backend/app/routes/scraping.py

```
@@ -441,6 +442,52 @@ async def quick_scrape_visitopatija(
+
+        background_tasks.add_task(run_vv_scrape)
+
+        return ScrapeResponse(
+            status="accepted",
+            message=f"VisitVarazdin scraping task started in background for {request.max_pages} pages",
+            task_id=task_id,
+        )
+    except Exception as e:
+        raise HTTPException(
+            status_code=500, detail=f"Failed to start VisitVarazdin scraping: {str(e)}"
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)